tool/update_version.dart

```diff
@@ -420,7 +420,7 @@ class AutoUpdateCommand extends Command {
           throw 'Failed to determine the newVersion.';
         }
     }
-    print('Updating from $currentVersion to $newVersion');
+    print('Update from $currentVersion to $newVersion');
```

Comment:
nit: I would leave as "Updating" since this log indicates that the action is in progress.

Reply:
<img width="368" alt="Screenshot 2023-02-27 at 10 46 40 AM" src="https://user-images.githubusercontent.com/12115586/221654978-b20c0e37-8461-430b-a936-926a4d09aa92.png">

Reply:
Do we want 'Updating' in this list instead of 'Update'?

Reply:
Changed to `Bump version from... `